test/integration/dependency.bats

```diff
@@ -32,15 +32,62 @@ function teardown() {
 	[[ "${output}" == *'"Name": "node-1"'* ]]
 }
 
-@test "links dependency" {
+@test "shared volumes(rw) dependency" {
```

Comment:
Can we squash `shared volumes`, `shared volumes(rw)` and `shared volumes(ro)` tests together?

We could just add a container `b3` and `b4` to `shared volumes` testing ro and rw.

The reason I'm asking is because every single test we add slows down the integration tests and since those are very similar they could be added to the same one